docs/API.md

```diff
@@ -609,6 +608,44 @@ Returns a unix time in seconds, like `,"time":1493426328`.
 Returns an empty string. This function is used when the `timestamp` option
 is set to `false`.
 
+<a id=".destination"></a>
+# .destination(dest?)
+
+Create a pino destination.
+It returns a stream-like object with significantly more throughput than a
+standard Node.js stream.
+
+```js
+const pino = require('pino')
+const logger = pino(pino.destination('./my-file'))
+const logger2 = pino(pino.destination())
+```
+
+`dest` could be either a file or a file descriptor. If it is omitted, it
+will be `process.stdout.fd`.
+
+The default `stream` is a destination.
+
+It is based on [`sonic-boom`](https://github.com/mcollina/sonic-boom).
```

Comment:
Pino destinations are based on

or 

The `pino.destination` API is implemented on `sonic-boom`